build/Dockerfile

```diff
@@ -1,12 +1,30 @@
-FROM golang:1.13.8 AS build
-
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# NOTE: using registry.access.redhat.com/rhel8/go-toolset does not work (user is requested to use registry.redhat.io)
+# NOTE: using registry.redhat.io/rhel8/go-toolset requires login, which complicates automation
+# NOTE: since updateBaseImages.sh does not support other registries than RHCC, update to RHEL8
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
```

Comment:
can you use ubi8 go-toolset here instead, to be consistent with https://github.com/che-incubator/kubernetes-image-puller/pull/30/files#diff-ebacf6f6ae4ee68078bb16454b23247dR12 ?

Reply:
is it standardized somehow? why the different image is used in the che-operator?

Reply:
used to be that there was no good rhel8 / ubi8 golang builder image. Seems that has since changed with 1.13 release

So... yes, we can switch them all over to use ubi8 golang 1.13.